kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -58,11 +58,11 @@
             <thead>
               <tr>
                 <th class="col-checkbox">
-                  <input
-                    type="checkbox"
+                  <k-checkbox
                     :checked="allExercisesWithinCurrentTopicSelected"
-                    :indeterminate.prop="someExercisesWithinCurrentTopicSelected"
-                    @change="changeSelection">
+                    :indeterminate="someExercisesWithinCurrentTopicSelected"
+                    @change="changeSelection"
+                  />
                 </th>
                 <th class="col-title">{{ $tr('selectAll') }}</th>
```

Comment:
Need remove the "Select all" label from inside `<th>`, and set the checkbox's `showLabel` to `true`:

* To a screenreader, they'll hear "Select all" read twice - one for the checkbox label and once for the column header
* The label isn't clickable

(you can use `colspan="2"` as I showed above to help with the spacing)